examples/bri-3/src/bri/identity/bpiSubjects/capabilities/createBpiSubject/createBpiSubjectCommand.handler.ts

```diff
@@ -0,0 +1,22 @@
+import { CommandHandler, ICommandHandler } from "@nestjs/cqrs";
+import { BpiSubjectAgent } from "../../agents/bpiSubjects.agent";
+import { CreateBpiSubjectCommand } from "./createBpiSubject.command";
+
+@CommandHandler(CreateBpiSubjectCommand)
+export class CreateBpiSubjectCommandHandler implements ICommandHandler<CreateBpiSubjectCommand> {
+  constructor(private agent: BpiSubjectAgent) {}
+
+  async execute(command: CreateBpiSubjectCommand) {
+    const { name, desc, publicKey } = command;
+
+    this.agent.throwIfCreateBpiSubjectInputInvalid(name, desc, publicKey);
+
+    const newBpiSubject = this.agent.createNewExternalBpiSubject(name, desc, publicKey);
+    
+    // TODO: Generic map of domain model to entity model
+    // this.orm.store(newBpiSubject);
+
+    // TODO: Response DTO
+    return true;
```

Comment:
is this return needed for commands?

Reply:
I just added for the case of the example. I do not think it is obligatory. I believe nestjs would by default return 200 in case no exceptions thrown (this needs to be validated in nestjs docs)